mu4/scenes/playback/internal/playbackcontroller.h

```diff
@@ -50,6 +52,8 @@ class PlaybackController : public IPlaybackController, public actions::Actionabl
     float playbackPosition() const override;
     async::Channel<uint32_t> midiTickPlayed() const override;
 
+    void playElementOnClick(const domain::notation::Element* e) override;
```

Comment:
Let's just call it playElement?

Reply:
At first I called it that.
But there is a setting to play elements on click or not.
So I decided to call this method `playElementOnClick`
And in the future, if necessary, add a simple `playElement` method, which should not depend on any settings, but should certainly play the element.